### PR TITLE
chore: remove tmuxinator

### PR DESCRIPTION
**Pull request by justinmoon**
At this point mprocs has fully replaced tmuxinator as our local dev environment of choice. The last remaining problem was that multi-line text selection wasn't working very well but AFAIK that's fixed now.

Closes https://github.com/fedimint/fedimint/issues/2426

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -395,7 +393,6 @@
                       if [ "$(tmux show-options -A default-command)" == 'bla' ]; then
                         echo
                         >&2 echo "⚠️  tmux's 'default-command' not set"
-                        >&2 echo " ️  Please add 'set -g default-command \"\''${SHELL}\"' to your '$HOME/.tmux.conf' for tmuxinator test setup to work correctly"
                       fi
                     fi
```
Comment (justinmoon) [inline flake.nix:397]: @dpc I'm not sure exactly what "tmux" this section does. Please advise.

**Comment by dpc** [comment]
> The last remaining problem was that multi-line text selection wasn't working very well but AFAIK that's fixed now.

Are we already running the newest `mprocs` with that fix?

**Review by douglaz — DISMISSED** [review]
I like tmuxinator, please keep it :cry:

**Comment by dpc** [comment]
BTW. How do I copy&paste in mprocs now? :D

**Comment by m1sterc001guy** [comment]
> BTW. How do I copy&paste in mprocs now? :D

Selecting the text then "c" worked for me

**Comment by justinmoon** [comment]
I will re-work this PR to keep tmuxinator but recommend mprocs as the default in `dev-running.md` tutorial

**Comment by dpc** [comment]
> Selecting the text then "c" worked for me

Oh, I get it. I have to use the mouse to select the text. A bit ew, but I guess it works.

**Comment by justinmoon** [comment]
Updated. Doesn't remove tmuxinator, but recommends mprocs in the docs. Also remove some old tmuxinator cruft like unused shell scripts and docs.

**Review by dpc — APPROVED** [review]

**Pull request merged**